src/interactive/azext_interactive/azclishell/app.py

```diff
@@ -98,7 +98,7 @@ def __init__(self, cli_ctx, style=None, completer=None,
             self.completer = AzCompleter(self, None)
             self.lexer = None
         self.history = history or FileHistory(os.path.join(self.config.get_config_dir(), self.config.get_history()))
-        os.environ[ENV_ADDITIONAL_USER_AGENT] = 'AZURECLISHELL/' + VERSION
+        os.environ[_ENV_ADDITIONAL_USER_AGENT] = 'AZURECLISHELL/' + VERSION
```

Comment:
If `AZURE_HTTP_USER_AGENT` is already set, this will overwrite it.

Reply:
Good point! Changed as requested.